R/defs.bzl

```diff
@@ -326,7 +326,8 @@ def _build_impl(ctx):
 
     transitive_tools = (library_deps["transitive_tools"] + _executables(ctx.attr.tools))
 
-    return [DefaultInfo(files=depset(output_files)),
+    return [DefaultInfo(files=depset(output_files),
+                        runfiles=ctx.runfiles(package_files, collect_default=True)),
```

Comment:
Why do we need to take runfiles with us here?

Reply:
Because of r_image, which I am going as you suggested to get to rules_docker instead.  The dep_layer/app_layer mechanism they use rely on runfiles by default, and there is basically only one rule that override the system, which is node.js, because node.js does not behave properly with some of their assumptions (especially around symlinks).

Reply:
If this really trips you, I can remove it here and use an aspect in rules_docker, but that would definitely complicate the code for implementing r_image properly.

Reply:
Leave it be if it makes things simple. I don't think it affects efficiency. I am going to put some effort into cleaning up this project this weekend, along with other things. At that point, I will look into what {lang}_image rules are doing, and may have more thoughts then.